addons/web/static/src/js/libs/pdfjs.js

```diff
@@ -18,3 +18,38 @@
 * Finally the option was choosed to modify the default value of this option directly in pdf.js as no hook worked in the
 * 'account_invoice_extract' module.
 */
+odoo.define('web.pdfjs.mixin', function (require) {
```

Comment:
I don't really like the name of this module. maybe rename it something like `web.pdfjs_customization`

Reply:
Since this is master, why not just use proper import/export? Then we don't need to name it.

Just need to place the file correctly, and I don't think it's a good idea to have custom code in the libs folder.

Reply:
About the `libs` folder, please note that it's the `web/static/src/libs/...` (customization of the libs) and not `web/static/lib/...` (the actual lib) :wink:

Reply:
I see, that `libs` folder name is confusing though. @ged-odoo would you consider renaming it to `lib_extensions` or something similar in master-wowl? As apparently it's already renamed there to include legacy in path.